assets/javascripts/vendor/govuk-template.js

```diff
@@ -63,16 +63,6 @@
   "use strict"
   var root = this;
   if(typeof root.GOVUK === 'undefined') { root.GOVUK = {}; }
-
-  GOVUK.addCookieMessage = function () {
-    var message = document.getElementById('global-cookie-message'),
-        hasCookieMessage = (message && GOVUK.cookie('seen_cookie_message') === null);
-
-    if (hasCookieMessage) {
-      message.style.display = 'block';
-      GOVUK.cookie('seen_cookie_message', 'yes', { days: 28 });
-    }
-  };
 }).call(this);
```

Comment:
@matthewmascord because the wrapper doesn't have anything else in it do you think we should remove it as well?

Reply:
yes, that would make sense wouldn't it 👍